src/WebJobs.Script/Description/PowerShell/PowerShellFunctionInvoker.cs

```diff
@@ -272,6 +273,16 @@ internal static List<string> GetModuleFilePaths(string rootScriptPath, string fu
                     PowerShellConstants.ModulesScriptFileExtensionPattern));
             }
 
+            if (Directory.Exists(rootModuleDirectory))
```

Comment:
Can we introduce a method to add a directory to `modulePaths` so we can combine this new code and the block above to eliminate duplication?